### ISSUE TITLE
configuration is not consistent with that in paper

### ISSUE THREAD

**Issue by UcanSee** [hologerry/SoCo#1]
Thanks for sharing such a good work! 
I am confused about the train configuration, base learning rate in this repository is 0.03 but 1.0 in paper, and weight decay in this repository is 0.000025 but 0.00001 in paper. Do these two different configurations result in significant performance changes?
